### ISSUE TITLE
[Android] Add polyline & polygon to the map 

### ISSUE THREAD

**Issue by sidorovroman** [mapsme/omim#556]
Hello! How can I add polylines and polygons to the map in the android application?

**Comment by deathbaba** [comment]
If you are asking about KML primitives, we don’t support polylines yet.

**Comment by sidorovroman** [comment]
Not KML, simple 2d lines and polygons, that are built just by coordinates (lat,lng)

**Comment by deathbaba** [comment]
Are you asking about API which can do it? Our current API supports only points on a map, but it can be improved to draw lines or polygons.

Btw, if you need to display lines on top of the map, you can import/put KML file with lines into MapsWithMe folder on a device.

**Comment by sidorovroman** [comment]
Yes, about API. Please tell me what I should improve for realize it?

ps: I need render lines and polygons dynamically, that's why using KML files is not best solution.

**Comment by deathbaba** [comment]
I would suggest to start with:
* map/mwm_url.cpp for api-url parsing
* map/track.hpp for displaying a track from KML
* Framework::ShowMapForURL for API etry-point
* API-clients entry points and examples here https://github.com/mapswithme <https://github.com/mapswithme>

I would also advise to use ```drape-2.0``` branch, it’s our new rendering engine and it will replace the old one in the master branch soon.
